<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>database</class>
 <widget class="QMainWindow" name="database">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Database</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>100</y>
      <width>158</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout4">
     <item>
      <widget class="QPushButton" name="helpButton">
       <property name="text">
        <string>Справка</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitButton">
       <property name="text">
        <string>Выход</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>42</y>
      <width>156</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout2">
     <item>
      <widget class="QLabel" name="idLabel">
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="idLineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>473</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="stateLabel">
       <property name="text">
        <string>Состояние COM-порта:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="settingsButton">
       <property name="text">
        <string>&amp;Настройки</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="connectButton">
       <property name="text">
        <string>&amp;Подключиться</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="disconnectButton">
       <property name="text">
        <string>&amp;Отключиться</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>70</y>
      <width>420</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout3">
     <item>
      <widget class="QLabel" name="enterLabel">
       <property name="text">
        <string>&amp;Введите фамилию:</string>
       </property>
       <property name="buddy">
        <cstring>enterLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="enterLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>&amp;Добавить в БД</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="synchButton">
       <property name="text">
        <string>&amp;Синхронизация</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>settingsButton</tabstop>
  <tabstop>connectButton</tabstop>
  <tabstop>disconnectButton</tabstop>
  <tabstop>idLineEdit</tabstop>
  <tabstop>enterLineEdit</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>synchButton</tabstop>
  <tabstop>helpButton</tabstop>
  <tabstop>exitButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>database</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>468</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>28</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
